components/event-bus/internal/knative/subscription/controller/subscription/reconcile.go

```diff
@@ -180,6 +205,13 @@ func (r *reconciler) reconcile(ctx context.Context, subscription *eventingv1alph
 			if err != nil {
 				return false, err
 			}
+			//TODO
+			log.Info("Delete knative sub A and set a new one")
```

Comment:
Please change `knative` to `Knative` in all the logs to make it consistent and don't shorten the words like `sub`. It should be easily understandable.

Reply:
This was meant to be removed as well